### DISCUSSION TITLE
Etherscan Contract Verification Not Working

### DISCUSSION THREAD

**Discussion by dino-rodriguez** [foundry-rs/foundry#4671] [Help]
Hello,

I am trying to verify my contract via `forge create` by adding the appropriate flags. When it gets to the verification step, I get:

```
Submitted contract for verification:
        Response: `OK`
        GUID: `wt28zi8xsmzbgnyeryvtau2gbb8k5yepwi1jzdwjrzmfdekrtf`
        URL:
        https://sepolia.etherscan.io/address/0xbaca359e5291d07401d9d4be4c5eb127febd8797
Contract verification status:
Response: `NOTOK`
Details: `Pending in queue`
Contract verification status:
Response: `NOTOK`
Details: `Fail - Unable to verify. Solidity Compilation Error: Source "node_modules/@openzeppelin/contracts/utils/Context.sol" not found: File not found. Searched the following locations: "".`
Contract failed to verify.
```

Does anyone know how to fix this? It seems it's an issue with me using the openzeppelin library via node modules.

I can compile, deploy, and transact with my contract locally using `anvil`, so it's not a compilation issue (I mean the contract is successfully deployed, it just fails verification).

**Comment by 0xV4L3NT1N3** [comment]
gm @dino-rodriguez, this seems to be an issue with the location of your OpenZeppelin imports, either being missing or placed in the wrong directory. 

Generally if you're able to run `forge build` without any errors, this should succesfully be submitted to Etherscan too via the `--verify` command.

Your contract does appear to be [verified](https://sepolia.etherscan.io/address/0xbaca359e5291d07401d9d4be4c5eb127febd8797#code) now, leaving a reference for future users!
